src/statistics.cc

```diff
@@ -13,16 +13,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "benchmark/benchmark.h"
-
-#include <algorithm>
-#include <cmath>
-#include <numeric>
-#include <string>
-#include <vector>
-#include "check.h"
 #include "statistics.h"
 
+#include <algorithm>  // for max, nth_element, count_if
+#include <cmath>      // for sqrt
+#include <iosfwd>     // for size_t
+#include <map>        // for map, _Rb_tree_iterator, operator==
```

Comment:
`_RB_tree_iterator, operator==` are bad tool generated comments and should be removed.